src/pages/Revisao/RevisarAuditarInscricao/RevisarAuditarInscricao.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Typography } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import FormInscricao from "../../Edital/Inscricao/Components/FormInscricao";
+import { IDetalhesInscricao, IRevisarAuditar } from "../Interfaces";
+import { IInscricaoData } from "../../Edital/Inscricao/Interfaces";
+import { patchAuditarInscricao, patchRevisarInscricao } from "../Service";
+
+interface IProps {
+  editalId: number;
+  inscricaoId: number;
+  isAuditoria: boolean;
+  dadosInscricao: IDetalhesInscricao;
+  setInscricaoError: (error: boolean) => void;
+}
+
+export default function RevisarAuditarInscricao({
+  editalId,
+  inscricaoId,
+  isAuditoria,
+  dadosInscricao,
+  setInscricaoError,
+}: IProps) {
+  const navigate = useNavigate();
+
+  const actionAfterRequestSuccess = (
+    _: number // eslint-disable-line @typescript-eslint/no-unused-vars
+  ) => {
+    const state = isAuditoria ? { auditoria: true } : { revisao: true };
+    navigate(`/edital/${editalId}/inscritos`, { state });
+  };
+
+  const submitRequest = (inscricaoData: IInscricaoData) => {
+    const payload: IRevisarAuditar = {
+      id: inscricaoData.id_inscricao,
+      nota_final: inscricaoData.nota_final,
+      observacao: inscricaoData.observacao_professor,
+    };
+
+    return (
+      isAuditoria
```

Comment:
e se n for nem auditor nem professor?
Essa rota ta protegida de aluno?

Reply:
já vi q sim hehe
boa

Reply:
hihihih